lib/driver-helper.js

```diff
@@ -528,3 +528,23 @@ export async function dismissAlertIfPresent( driver ) {
 		return false;
 	}
 }
+
+export async function clear_cache( driver ) {
+	let url = await driver.getCurrentUrl();
+	if ( url.startsWith( 'data:' ) === false && url !== 'about:blank' ) {
+		return;
+	}
+	// Clear the cookies and cache for the ChromeDriver instance
+	// navigate to the settings page
+	await driver.get( 'chrome://settings/clearBrowserData' );
+	const clearCacheSelector = by.css( '* /deep/ #clearBrowsingDataConfirm' );
```

Comment:
There are serialized calypso state data in the local storage, so I'm wondering if we should make sure all the checkboxes has been checked before we let the webdriver click the button. e.g.
```
    const optionCheckboxesSelector = by.css( '* /deep/ #checkbox' );

    // wait for the checkboxes to appear
    await isEventuallyPresentAndDisplayed( driver, optionCheckboxesSelector );

    // toggle all the checkboxes
    await clickWhenClickable( driver, optionCheckboxesSelector );
```
before this line.

For example, in my case running the test `'Sign up for a domain only purchase coming in from wordpress.com/domains in EUR currency @parallel'` at line and then `'Sign up for a site on a business paid plan w/ domain name coming in via /create as business flow in CAD currency @parallel'` will make the later to fail. There seems to be some redux state left after running the former, and causes visiting `/start` being redirected to `/start/business/themes`.

Theoretically speaking `window.localStorage.clear()` should have cleaned all of them, but it doesn't seem that way.

Reply:
Hmm I've been giving this a try today and I seem to see a little more improvement, still working on it, look out for a few more commits as I work through it a bit more

> Theoretically speaking window.localStorage.clear() should have cleaned all of them, but it doesn't seem that way.

Yeah it's weird that something seems to be still left over, I'm wondering if somehow the test isn't waiting for the script to finish executing 🤷🏽‍♂️